infrastructure/running-a-node/setup-secure-wss.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Setup Secure WebSocket
+description: Instructions on enabling SSL for your node and setting up a secure WebSocket proxy server using nginx for remote connections.
+---
+
+# Setup Secure WebSocket
+
+## Introduction
+
+Ensuring secure WebSocket communication is crucial for maintaining the integrity and security of a Polkadot or Kusama node when interacting with remote clients. This guide walks you through setting up a secure WebSocket (WSS) connection for your node by leveraging SSL encryption with popular web server proxies like nginx or Apache. By the end of this guide, you'll be able to secure your node's WebSocket port, enabling safe remote connections without exposing your node to unnecessary risks.
+
+!!!info
+    The following instructions are for UNIX-based systems.
+
+## Secure a WebSocket Port
+
+You can convert a non-secured WebSocket port to a secure WSS port by placing it behind an SSL-enabled proxy. This approach can be used to secure a bootnode or RPC server. The SSL-enabled apache2/nginx/other proxy server redirects requests to the internal WebSocket and converts it to a secure (WSS) connection. You can use a service like [LetsEncrypt](https://letsencrypt.org/){target=\_blank} to obtain an SSL certificate.
+
+### Obtain an SSL Certificate
+
+You can follow the [LetsEncrypt](https://letsencrypt.org/){target=\_blank} instructions for your respective web server implementation to get a free SSL certificate:
```

Comment:
```suggestion
LetsEncrypt suggests using the [Certbot ACME client](https://letsencrypt.org/getting-started/#with-shell-access/){target=\_blank} for your respective web server implementation to get a free SSL certificate:
```
I found it confusing when these links didn't open to LetsEncrypt, so added some language to clarify where the user can expect to end up